academic_observatory_workflows/openalex_telescope/openalex_telescope.py

```diff
@@ -69,7 +69,7 @@
 )
 
 TEMP_TABLE_DESCRIPTION = "Temporary table for internal use. Do not use."
-UPSERT_BYTE_LIMIT = int(2.5 * 2**40)
+UPSERT_BYTE_LIMIT = int(3 * 2**40)
```

Comment:
I forget, does AO have a config file? If so maybe we could supply the byte limit through it as it seems to be changing fairly frequently?

Reply:
True, if I make it as an argument for the workflow with a default value, then we can specify it in the config file.